app/src/backend/mocks/createCollection.ts

```diff
@@ -0,0 +1,55 @@
+import firebaseApp from "../../firebase";
```

Comment:
<img width="1212" alt="CleanShot 2024-05-04 at 13 27 56@2x" src="https://github.com/requestly/requestly/assets/16779465/9e809afb-9105-4633-a4d4-48b72270b34e">

When a mock within a collection is starred, show only the pinned and the collection in the pinned tab. Don't show the rest of the unstarred mocks

Reply:
Last Modified not centered

Reply:
<img width="556" alt="CleanShot 2024-05-04 at 13 37 20@2x" src="https://github.com/requestly/requestly/assets/16779465/5d89df49-9529-49b1-9d74-ac11ee84bdd3">


- Search for Mock within collection : Only that mock and collection visible
- Search for Collection: Group and all the mocks within it visible